pkg/noun/jets/b/mate.c

```diff
@@ -0,0 +1,30 @@
+/// @file
+
+#include "jets/q.h"
+#include "jets/w.h"
+
+#include "noun.h"
+
+
+  u3_noun
+  u3qb_mate(u3_noun a,
+            u3_noun b)
+  {
+    if ( u3_nul == b ) {
+      return u3k(a);
+    } else if ( u3_nul == a ) {
+      return u3k(b);
+    } else if ( c3y == u3r_sing(a, b) ) {
```

Comment:
To avoid a jet mismatch, you have to do this comparison explicitly against `u3t(a)` and `u3t(b)`.

Reply:
Yes, you're quite correct. Thank you!